### ISSUE TITLE
draw.io diagram editor can not be closed

### ISSUE THREAD

**Issue by fschrempf** [owncloud/web#10439]
## Describe the bug

Using the `drawi.io` diagram editor in the web UI, closing the editor from within the editor itself is not possible.

## Steps to reproduce
1. Run oCIS 4.0.5 as per documentation
2. Enable the `draw.io` app by editing the `config.json`
3. Open a `*.drawio` file in the web UI
4. Use the `Exit` Button to close the editor

## Expected behavior
The diagram editor is closed.

## Actual behavior
The diagram editor stays open and the browser console gives a warning: `Scripts may close only the windows that were opened by them.`

## Setup

`config.json`

```json
[...]
  "external_apps": [
    {
      "id": "draw-io",
      "path": "web-app-draw-io",
      "config": {
        "url": "https://embed.diagrams.net",
        "theme": "minimal"
      }
    }
  ]
```

**Comment by JanAckermann** [comment]
Thanks for your report, we've fixed this in the upcoming version ocis-5.0.0

**Comment by RupertBarrow** [comment]
Hi @JanAckermann , could you point me to the fix in the source code, please, or the issue/PR which fixes it ? Thanks
